standalone/gui/application.cpp

```diff
@@ -0,0 +1,245 @@
+                                 hstretch(0.5, margin({ 10., 10., 10., 10. }, pane("Features", make_features(), 1.0F))),
+                                 margin({ 10., 10., 10., 10. }, pane("Player", make_player()))
+                             )
+                         )
+               )
+        );
+}
+
+void application::dial_value(int dial_index, double val, view &view_) {
+
+    dials[dial_index]->dial_base::value(val);
+    view_.refresh(*dials[dial_index]);
+
+    labels[(dial_index+3)]->set_text(std::to_string(val));
+    view_.refresh(*labels[(dial_index+3)]);
+}
+
+void application::slider_value(int slider_index, double val, view &view_) {
+
+    vertical_sliders[slider_index]->slider_base::value(val);
+    view_.refresh(*vertical_sliders[slider_index]);
+
+    labels[slider_index]->set_text(std::to_string(val));
+    view_.refresh(*labels[slider_index]);
+}
+
+void application::link_control(int index, view &view_) {
+    if(index <= 2){
+
+        vertical_sliders[index]->on_change =
+            [index, &view_, this](double val){
+
+              labels[index]->set_text(std::to_string(val));
+              view_.refresh(*labels[index]);
+
+              if(index == 0){
+                  dial_value(1, val, view_);
+                  dial_value(3, val, view_);
+                  dial_value(4, val, view_);
+              }
+
+              if(index == 1){
+                  dial_value(0, val, view_);
+                  dial_value(2, val, view_);
+                  dial_value(3, val, view_);
+                  dial_value(4, val, view_);
+                  dial_value(5, val, view_);
+              }
+            };
+    }
+
+    int dials_index = index + 3;
+
+    dials[index]->on_change =
+        [index, dials_index, &view_, this](double val){
+
+          labels[dials_index]->set_text(std::to_string(val));
+          view_.refresh(*labels[dials_index]);
+
+          if(index == 0 || index == 2 || index == 3 || index == 4 || index == 5){
+              slider_value(1, val,view_);
+              dial_value(2, val, view_);
+              dial_value(3, val, view_);
+              dial_value(4, val, view_);
+              dial_value(5, val, view_);
+          }
+
+          if(index == 1 || index == 3 || index == 4){
+              slider_value(0, val,view_);
+              dial_value(3, val, view_);
+              dial_value(4, val, view_);
+          }
+        };
+}
+void application::link_controls(view &view_) {
+    link_control(0, view_);
+    link_control(1, view_);
+    link_control(2, view_);
+    link_control(3, view_);
+    link_control(4, view_);
+    link_control(5, view_);
+}
+
+// Initialize the application
+// The application will stop when the window is closed
+void application::init_application(cycfi::elements::app &_app) {
+
+    _window.on_close = [&_app]() { _app.stop(); };
+
+    _view.content(
+        make_application(),
+        _background
+    );
+
+    link_controls(_view);
+
+    _app.run();
```

Comment:
Está mostrando isso como código novo, o que é estranho pois isso já está alterado no PR#1. Talvez esteja desfazendo algo do PR#1. Bom conferir.

Reply:
Dei um pull no master para garantir, mas ele ainda continua mostrando essas alterações.

Reply:
Se estiver usando o master como referência do último checkout, talvez é porque ele esteja interpretando como uma alteração no master mesmo que é razoável. Nesse caso, é só dar rollback que vai levar para a versão do master.

Reply:
Fiz o checkout para a versão que está no master, mas essa alteração continua aparecendo. Pesquisei algumas soluções e parece que isso acontece porque os commits do PR#1 ainda estão aparecendo aqui. Testei forçar um rebase em outro repositório e funciona. Acho que ele removeria todos os commits que já foram para o merge. Nesse caso, não há problemas com essa solução, certo?

Reply:
Estranho. Porque lá em baixo está dizendo "This branch has no conflicts with the base branch". Ou o base branch não é o master ou ele está entendendo como alterações no master. De qualquer forma, não precisa preocupar tanto com o que ele está entendendo. Só vamos ficar atentos para ver se o PR#2 não está desfazendo nada do PR#1 e aceitamos, independente do que ele está entendendo.

Reply:
Entendi. Conferi com o meu último commit e ele realmente não está desfazendo o que está no master.